analysis/db/us_map/choroplethMap.py

```diff
@@ -34,7 +34,10 @@
 molten_df['date_iso'] = pd.to_datetime(molten_df['date'], format="%m/%d/%y")  # change date to ISO8601 standard format
 
 fips = molten_df['fips_str'].tolist()
-# max_val = molten_df['value'].max()
+
+# plt.show()
+# color_map = plt.cm.get_cmap('viridis')
+# reversed_viridis = color_map.reversed()
 
 
 fig = px.choropleth(molten_df,
```

Comment:
I created a dataframe that was just a subset of a particular date, and then used that subseted dataframe to plot the figure 

```python
plot_data = molten_df[molten_df.date_iso == '2020-02-01']
fig = px.choropleth(plot_data,
                    geojson=counties,
                    locations=plot_data.fips_str,
                    color='value',
                    #animation_frame='date',
                    hover_data=['State', 'value'],
                    color_continuous_scale='viridis_r',
                    range_color=(0, 300),
                    scope="usa",
                    title='Confirmed cases',
                    labels={'value': 'confirmed cases'}
                    )
```